packages/esm-patient-medications-app/src/drug-order-basket-panel/order-basket-item-tile.component.tsx

```diff
@@ -25,69 +25,76 @@ export default function OrderBasketItemTile({ orderBasketItem, onItemClick, onRe
-            {orderBasketItem.patientInstructions && <>&mdash; {orderBasketItem.patientInstructions}</>}
           </span>
-        </span>
-        <br />
-        <span className={styles.label01}>
-          <span className={styles.indicationLabel}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
-          <span className={styles.dosageInfo}>
-            {!!orderBasketItem.indication ? orderBasketItem.indication : <i>{t('none', 'None')}</i>}
+          <br />
+          <span className={styles.label01}>
+            <span className={styles.indicationLabel}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
+            <span className={styles.dosageInfo}>
+              {!!orderBasketItem.indication ? orderBasketItem.indication : <i>{t('none', 'None')}</i>}
+            </span>
+            {!!orderBasketItem.orderError && (
+              <>
+                <br />
+                <span className={styles.orderErrorText}>
+                  <WarningIcon size={16} /> &nbsp;{' '}
+                  <span className={styles.label01}>{t('error', 'Error').toUpperCase()}</span> &nbsp;
+                  {orderBasketItem.orderError.responseBody?.error?.message ?? orderBasketItem.orderError.message}
+                </span>
+              </>
+            )}
           </span>
-          {!!orderBasketItem.orderError && (
-            <>
-              <br />
-              <span className={styles.orderErrorText}>
-                <WarningIcon size={16} /> &nbsp;{' '}
-                <span className={styles.label01}>{t('error', 'Error').toUpperCase()}</span> &nbsp;
-                {orderBasketItem.orderError.responseBody?.error?.message ?? orderBasketItem.orderError.message}
-              </span>
-            </>
-          )}
-        </span>
+        </div>
+        <Button
+          className={styles.removeButton}
+          kind="ghost"
+          hasIconOnly={true}
+          renderIcon={(props: ComponentProps<typeof TrashCanIcon>) => <TrashCanIcon size={16} {...props} />}
+          iconDescription={t('removeFromBasket', 'Remove from basket')}
+          onClick={() => {
+            shouldOnClickBeCalled.current = false;
+            onRemoveClick();
+          }}
+          tooltipPosition="left"
+        />
       </div>
-      <Button
-        className={styles.removeButton}
-        kind="ghost"
-        hasIconOnly={true}
-        renderIcon={(props: ComponentProps<typeof TrashCanIcon>) => <TrashCanIcon size={16} {...props} />}
-        iconDescription={t('removeFromBasket', 'Remove from basket')}
-        onClick={() => {
-          shouldOnClickBeCalled.current = false;
-          onRemoveClick();
-        }}
-        tooltipPosition="left"
+      <ExtensionSlot
+        name="order-item-additional-info-slot"
+        state={{ orderItemUuid: orderBasketItem.drug.uuid }}
+        className={styles.additionalInfoContainer}
```

Comment:
Instead of introducing the Extension Slot multiple times, can we make a common component and import that component directly?